yt/visualization/volume_rendering/off_axis_projection.py

```diff
@@ -325,7 +325,10 @@ def temp_weightfield(field, data):
             return temp_weightfield
 
         data_source.ds.field_info.add_field(
-            weightfield, sampling_type="cell", function=_make_wf(item, weight)
+            weightfield,
+            sampling_type="cell",
+            function=_make_wf(item, weight),
+            units="",
```

Comment:
why this change ?

Reply:
For some reason `pre-commit` asked for it

Reply:
I mean, the formatting is just `black` being triggered by the line getting long, but you're also adding a `units=""` argument, and that's what confuses me.

Reply:
The `units` here is dimensionless, so I'm just making it explicit here.